atomic_reactor/util_pulp.py

```diff
@@ -0,0 +1,232 @@
+"""
```

Comment:
For the Koji equivalent of this we named it `koji_util.py`. We should be consistent, one way or the other.